tests/phpunit/Generator/Synapse/ImportFull/FromAbsGeneratorTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\CustomQueryManagerApp\Tests\Generator\Synapse\ImportFull;
+
+use Keboola\CustomQueryManagerApp\Generator\Synapse\ImportFull\FromAbsGenerator;
+use PHPUnit\Framework\TestCase;
+
+class FromAbsGeneratorTest extends TestCase
+{
+    public function testGenerate(): void
+    {
+        $generator = new FromAbsGenerator();
+
+        $columns = [
+            'column1',
+            'column2',
+        ];
+        $primaryKeys = [
+            'column1',
+        ];
+        $queries = $generator->generate($columns, $primaryKeys);
+
+        /** @codingStandardsIgnoreStart */
+        $expected = [
+            "CREATE TABLE {{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp') }} ([column1] NVARCHAR(4000), [column2] NVARCHAR(4000)) WITH (DISTRIBUTION = ROUND_ROBIN,CLUSTERED COLUMNSTORE INDEX)",
```

Comment:
To je špatně protože stage tabulka je temp takže začíná #

Reply:
zmeneno na `id(stageTableName)`